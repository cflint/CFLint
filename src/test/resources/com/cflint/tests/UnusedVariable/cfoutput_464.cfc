<cfcomponent>
<cffunction name="test" access="public" returnType="any" output="true">
	<cfscript>
		var resultText="";
	</cfscript>
	<cfset resultText="error">
	<cfsetting showdebugoutput="false" />
	<cfcontent type="text" reset="yes"><cfoutput>#resultText#<cfset foo=""/></cfoutput><cfabort>
</cffunction>
</cfcomponent>