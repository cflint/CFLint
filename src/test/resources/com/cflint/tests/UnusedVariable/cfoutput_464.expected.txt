{
  "version" : "",
  "timestamp" : 1568216331,
  "issues" : [ ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ ],
    "countBySeverity" : [ ]
  }
}